website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -80,6 +80,51 @@ Use this variant only when you need to put interactive elements such as a button
 
 ![Example of toggle content](/assets/components/accordion/toggle-content-interactive.png =800x*)
 
+## Sizes
+
+Our accordion comes in three sizes: `small`, `medium`, and `large`. We recommend using `medium`, but use whichever size best fits your UI’s needs.
+
+![Image here of different sizes - only card variant](/assets/components/accordion/accordion-size-range.png)
+
+!!! Dont
+
+The size is set at the Accordion (group) level, so accordionItem sizes are not interchangeable.
+
+![Image here of DONT change sizes - only card variant](/assets/components/accordion/accordion-size-Dont.png)
+!!!
+
+## Types
+
+Our accordion has two container types: `card` and `flush`.
+
+### Card
+
+The `card` variant is the default and recommended type, creating visual separation between accordion sections and other UI elements. It helps organize content blocks and distinguish between sections within tabs. Unlike `Cards`/`Tiles` or `Tables`, the `card` variant groups multiple expandable sections for flexible content management.
+
+!!! Do
+
+![image of DO example for the card variant](/assets/components/accordion/accordion-card-type-example.png)
+!!!
+
+### Flush
+
+Use the flush variant for tight spaces. For example, use it within other containers like cards and flyouts, or in sidebars where space is limited.
+
+!!! Do
+
+![image of DO example for the Flush variant](/assets/components/accordion/accordion-flush-example.png)
+!!!
+
+### Static AccordionItems
```

Comment:
If we are adding Static AccordionItems part of the `types` section, we may want to call it out on line 98 with the rest of the types, but distinguish this as an accordionItem property only.

Reply:
Updated line 75 with Static as an AccordionItem

Reply:
I left a comment about this above but I'm not sure I agree with this. To me it makes sense to have the static content as it's own section because it isn't technically a type (at least how we define it). I might recommend:

- Type
  - Card
  - Flush
- Static AccordionItems

Which matches how the properties are structured in the component.

Reply:
`isStatic` is a boolean (true/false) property within the code: https://hds-website-git-accordion-documentation-hashicorp.vercel.app/components/accordion?tab=code#component-api

Reply:
Btw, did we consider calling the `type` property, `containerType`? That could make the purpose & usage clearer.

I would consider the two types/containerTypes to primarily be visual variations, vs "static" (`isStatic=true`) which primarily affects the functionality. (Any visual affect is secondary and could presumedly be combined with a containerType of either "card" or "flush".)

Reply:
+1 to Jory's comment. I also commented on the heading structure, which needs to be revisited.

Reply:
@KristinLBradley originally we did, but my memory is slipping for the change in naming to just "Type"

Reply:
here is [some background on the choice](https://hashicorp.slack.com/archives/C025N5V4PFZ/p1718715573875349)
tl;dr the `flush` variant doesn't only affect the container, but the trigger as well, as the style (border, shadow) is applied to the whole `AccordionItem`. it also aligns with how we named such argument in other components